components/assetlist/stories/assetlist.test.js

```diff
@@ -1,4 +1,73 @@
 import { Variants } from "@spectrum-css/preview/decorators";
 import { Template } from "./template.js";
 
-export const AssetListGroup = Variants({ Template });
+const items = [
+	{
+		image: "example-ava.png",
+		label: "Shantanu.jpg",
+		ariaLabelledBy: "assetitemlabel-1",
+		checkboxId: "checkbox-1"
+	},
+	{
+		iconName: "Document",
+		iconSet: "workflow",
+		label: "Resource allocation documentation should truncate",
+		ariaLabelledby: "assetitemlabel-2",
+		checkboxId: "checkbox-2",
+	},
+	{
+		iconName: "Folder",
+		iconSet: "workflow",
+		label: "Front-end projects",
+		ariaLabelledby: "assetitemlabel-3",
+		checkboxId: "checkbox-3",
+		isBranch: true,
+	},
+	{
+		iconName: "Folder",
+		iconSet: "workflow",
+		label: "Downloads",
+		ariaLabelledby: "assetitemlabel-4",
+		checkboxId: "checkbox-4",
+		isBranch: true,
+	},
+];
+export const AssetListGroup = Variants({
+	Template,
+	testData: [
+		{
+			testHeading: "Default",
+			items,
+		},
```

Comment:
🤯 Ok, I'm realizing there's a lot I don't understand about this component... in the default version, I see checkboxes on the right:

<img width="259" height="223" alt="image" src="https://github.com/user-attachments/assets/566443ca-df42-478e-947c-62c49c8a2156" />

And then for Selectable, they're on the left:
<img width="269" height="204" alt="image" src="https://github.com/user-attachments/assets/29f245db-18ca-417b-aa0c-73d20b654fd1" />

I'm also seeing this difference in the Miller columns stories/tests. Is that deliberate? Is this new? And if it's always been like this, would it be possible to maybe document to clarify?